runtime/CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ set(conditionalVersionString)
 configure_file(include/j9cfg.h.in j9cfg.h)
 configure_file(include/j9lib.h.in j9lib.h)
 configure_file(include/j9version.h.in j9version.h)
+configure_file(makelib/copyright.c.in copyright.c)
```

Comment:
Confirming that the intention is to garbage collect the `.ftl` versions of these files once cmake is the only way to build?

Reply:
correct